Makefile

```diff
@@ -88,7 +89,11 @@ clean:
 
 .PHONY: lint
 lint: my.env .docker-build
-	${DC} run --rm --no-deps base  /bin/bash ./docker/run_lint.sh
+	${DC} run --rm --no-deps base /bin/bash ./docker/run_lint.sh
+
+.PHONY: black
+lintfix: my.env .docker-build
+	${DC} run --rm --no-deps base /bin/bash ./docker/run_lint.sh --fix
```

Comment:
"lint" and "lintfix" are tools-agnostic names that sort of match what Tecken is doing and what "mach" does. We'll be able to use these across all the projects.